<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1263</width>
      <height>627</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/resources/resources/left.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="boton_adelante">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>170</y>
       <width>93</width>
       <height>297</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>297</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resource/resources/right.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_atras">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>93</width>
       <height>297</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>297</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-image: url(:/resource/resources/left.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>511</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(85, 0, 127);
font: 87 30pt &quot;Segoe UI Black&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Seleccionar usuario:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botonbarrido">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>40</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#E6FAFC;
font: 87 16pt &quot;Segoe UI Black&quot;;

background-color: rgb(85, 0, 127);</string>
     </property>
     <property name="text">
      <string>barrido:OFF</string>
     </property>
    </widget>
    <widget class="usercard" name="card1" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>232</width>
       <height>329</height>
      </rect>
     </property>
    </widget>
    <widget class="usercard" name="card2" native="true">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>130</y>
       <width>233</width>
       <height>329</height>
      </rect>
     </property>
    </widget>
    <widget class="usercard" name="card3" native="true">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>130</y>
       <width>232</width>
       <height>329</height>
      </rect>
     </property>
    </widget>
    <widget class="usercard" name="card4" native="true">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>130</y>
       <width>232</width>
       <height>329</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>usercard</class>
   <extends>QWidget</extends>
   <header>usercard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
